### PR TITLE
Create invoiceshelf.yml

### PR DESCRIPTION
**Pull request by gdarko**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/invoiceshelf.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+source_code_url: "https://github.com/InvoiceShelf/InvoiceShelf"
+description: "InvoiceShelf is an open-source software that helps you track expenses, payments & create professional invoices & estimates."
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
+  - Docker
+tags:
+  - Money, Budgeting & Management
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/invoiceshelf.yml:12]:
```suggestion
```
No need, this line is redundant since it's already set to false. 👍🏻

Comment (gdarko) [reply software/invoiceshelf.yml]: Done @kokomo123 - Thanks for pointing out.

File: software/invoiceshelf.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "InvoiceShelf"
+website_url: "https://invoiceshelf.com/"
+source_code_url: "https://github.com/InvoiceShelf/InvoiceShelf"
+description: "InvoiceShelf is an open-source software that helps you track expenses, payments & create professional invoices & estimates."
```
Comment (kokomo123) [inline software/invoiceshelf.yml:4]:
```suggestion
description: "InvoiceShelf is an open-source software that helps you track expenses, payments & create professional invoices & estimates (fork of Crater)."
```
image
as pointed out here

Comment (gdarko) [reply software/invoiceshelf.yml]: done @kokomo123 - Thanks.

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here, seems mature enough, pending other thoughts from collaborators. Thank you for your contribution!

File: software/invoiceshelf.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: "InvoiceShelf"
+website_url: "https://invoiceshelf.com/"
+source_code_url: "https://github.com/InvoiceShelf/InvoiceShelf"
+description: "InvoiceShelf is an open-source software that helps you track expenses, payments & create professional invoices & estimates (fork of Crater)."
```
Comment (Rabenherz112) [inline software/invoiceshelf.yml:4]: Shorten the description slightly to get rid of redundant information. 
```suggestion
description: "A software that helps you track expenses, payments & create professional invoices & estimates (fork of Crater)."
```

Comment (gdarko) [reply software/invoiceshelf.yml]: Done @Rabenherz112  - Thanks for the suggestion.

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**